flow_action_components/Summer18/Google/force-app/main/default/classes/GetGoogleSheetsData.cls

```diff
@@ -0,0 +1,180 @@
+            parsedColumn = 26 * parsedColumn + column.charAt(i) - 'a'.charAt(0) + 1;        
+        }
+        parsedColumn--;
+        Address result = new Address();
+        result.row = row;
+        result.column = parsedColumn;
+        return result;
+    }
+
+    class Address {
+        public Integer row;
+
+        public Integer column;
+    }
+
+    global class GoogleSheetsRequest {
+
+        @InvocableVariable(required=True label='Sheets File Name' description='Case-sensitive, no extension, name only')
+        global String SpreadsheetName;
+
+        @InvocableVariable(required=True label='#1 Direction' description='Allowed values: Address, Right, Below')
+        global String Direction1;
+
+        @InvocableVariable(required=True label='#1 Location' description='What value to search for. If related direction is set to "Address", the format should be Excel-like (e.g. "A1")')
+        global String Location1;
+
+        @InvocableVariable(label='#2 Direction' description='Allowed values: Address, Right, Below')
+        global String Direction2;
+
+        @InvocableVariable(label='#2 Location' description='What value to search for. If related direction is set to "Address", the format should be Excel-like (e.g. "A1")')
+        global String Location2;
+
+        @InvocableVariable(label='#3 Direction' description='Allowed values: Address, Right, Below')
+        global String Direction3;
+
+        @InvocableVariable(label='#3 Location' description='What value to search for. If related direction is set to "Address", the format should be Excel-like (e.g. "A1")')
+        global String Location3;
+
+        @InvocableVariable(label='#4 Direction' description='Allowed values: Address, Right, Below')
+        global String Direction4;
+
+        @InvocableVariable(label='#4 Location' description='What value to search for. If related direction is set to "Address", the format should be Excel-like (e.g. "A1")')
+        global String Location4;
+
+        @InvocableVariable(label='#5 Direction' description='Allowed values: Address, Right, Below')
+        global String Direction5;
+
+        @InvocableVariable(label='#5 Location' description='What value to search for. If related direction is set to "Address", the format should be Excel-like (e.g. "A1")')
+        global String Location5;
+    }
+
+    global class GoogleSheetsResponse {
+        
+        @InvocableVariable(label='Value #1')
+        global String Value1;
+        
+        @InvocableVariable(label='Value #2')
+        global String Value2;
+        
+        @InvocableVariable(label='Value #3')
+        global String Value3;
+        
+        @InvocableVariable(label='Value #4')
+        global String Value4;
+        
+        @InvocableVariable(label='Value #5')
+        global String Value5;
+
+        @InvocableVariable(label='Is Success' description='"True" if retrieval was successful, otherwise "False"')
```

Comment:
I prefer to make these status strings instead of booleans so you can add more states in the future if you need to